packages/inspector/src/components/EntityInspector/ActionInspector/PlayCustomEmoteAction/PlayCustomEmoteAction.tsx

```diff
@@ -32,13 +32,28 @@ const PlayCustomEmoteAction: React.FC<Props> = ({ value, onUpdate }: Props) => {
     [setPayload, value, onUpdate],
   );
 
-  const handleChangeSrc = useCallback(
+  const handleDrop = useCallback(
     (path: string) => {
       handleUpdate({ ...payload, src: path });
     },
     [payload, handleUpdate],
   );
 
+  const handleChangeSrc = useCallback(
+    (e: React.ChangeEvent<HTMLInputElement>) => {
+      const rawValue = e.target.value;
+
+      // Add basePath if it doesn't already have the basePath included
+      let processedPath = rawValue;
+      if (rawValue && files?.basePath && !rawValue.startsWith(files.basePath)) {
+        processedPath = `${files.basePath}/${rawValue}`;
+      }
```

Comment:
since this is repeating itself in the next 2 files, maybe we can move this out to a function that returns the proper path and call it from the action files